Cargo.toml

```diff
@@ -42,9 +42,20 @@ default-panic-hook = []
 # enabled by default.
 legacy-runtime = ["neon-runtime/neon-sys", "neon-build/neon-sys"]
 
-# Feature flag to enable the experimental N-API runtime. For now, this feature
+# Default N-API version. Prefer to select a minimum required version.
```

Comment:
Would this be a good place to link to definitive documentation on the N-API version numbers and what they included?